hive/src/test/java/org/apache/iceberg/hive/ScriptRunner.java

```diff
@@ -1,23 +1,22 @@
 /*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
  *
- *  Copyright 2004 Clinton Begin
```

Comment:
I am not sure about the header here.

Reply:
We need to keep the old header. Apache projects do not change copyright headers, except to add additional notes about modifications.

Reply:
Done